### PR TITLE
Rem 353

### PR DESCRIPTION
**Pull request by Ghinwa-Arab**
Move Users/sites tab from argus to react

### CODE REVIEW

File: src/pages/users/Windows/UsersWindow.js [hunk 1]
```diff
@@ -38,12 +40,15 @@ const UsersWindow = ({ labels, maxAccess, recordId }) => {
         <DocTypeTab labels={labels} maxAccess={maxAccess} storeRecordId={storeRecordId} />
       </CustomTabPanel>
       <CustomTabPanel index={4} value={activeTab}>
-        <RowAccessTab labels={labels} maxAccess={maxAccess} storeRecordId={storeRecordId} />
```
Comment (RebeccaFrem02) [inline src/pages/users/Windows/UsersWindow.js:41]: you can send only storeRecordId,

File: src/pages/users/Windows/UsersWindow.js [hunk 2]
```diff
@@ -38,12 +40,15 @@ const UsersWindow = ({ labels, maxAccess, recordId }) => {
...
+        <SitesTab labels={labels} maxAccess={maxAccess} recordId={storeRecordId} />
```
Comment (RebeccaFrem02) [inline src/pages/users/Windows/UsersWindow.js:43]: you can send only storeRecordId,

File: src/pages/users/Tabs/SitesTab.js [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+      }
+    })()
+  }, [recordId])
+
+  return (
+    <FormShell form={formik} infoVisible={false} isCleared={false}>
+      <VertLayout>
+        <Fixed>
+          <Grid container>
+            <Grid xs={4} item>
```
Comment (RebeccaFrem02) [inline src/pages/users/Tabs/SitesTab.js:208]: do we need these two Grids?

File: src/pages/users/Tabs/SitesTab.js [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+              <CustomTextField
```
Comment (RebeccaFrem02) [inline src/pages/users/Tabs/SitesTab.js:209]: couldn't you use the search from gridToolbar?

File: src/pages/users/Tabs/SitesTab.js [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+          site.accessLevelName = userAccess.accessLevelName
+        }
+      })
+
+      const data = siteViews.map((item, index) => ({
+        ...item,
+        id: index + 1
+      }))
+      formik.setValues({ rows: data })
+    } catch (error) {}
```
Comment (RebeccaFrem02) [inline src/pages/users/Tabs/SitesTab.js:125]: please use error windows for handling errors

File: src/pages/users/Windows/UsersWindow.js (hunk 2 above)
Comment (Ghinwa-Arab) [reply src/pages/users/Windows/UsersWindow.js:43]: how? i'm renaming it to recordId and using it in children as recordId

File: src/pages/users/Tabs/SitesTab.js (hunk 5 above)
Comment (Ghinwa-Arab) [reply src/pages/users/Tabs/SitesTab.js:125]: no need

**Pull request merged**